main.py

```
@@ -112,16 +208,275 @@ async def validate_and_sync_packs(bot, user_id):
+        else:
+            end = start.replace(month=start.month + 1)
+    return format_sqlite_datetime(start), format_sqlite_datetime(end)
+
+
+def check_creation_limit(telegram_id):
+    """Return (allowed: bool, remaining: int) for the user's current period."""
+    plan = get_user_plan(telegram_id)
+    limits = PLAN_LIMITS.get(plan, PLAN_LIMITS['free'])
+    period_type = limits['period']
+    max_creations = limits['creations']
+    period_start, period_end = _current_period_bounds(period_type)
```

Ruff (0.15.7)

[warning] Unpacked variable `period_end` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)